src/test/testdata/expected/lsl/charLiterals.lsl

```diff
@@ -0,0 +1,12 @@
+libsl "1.0.0";
+library simple;
+typealias Int = int32;
+typealias Char = char;
+
+automaton A : Int {
+    var ch1: Char = 'A';
+    var ch2: Char = 'a';
+    var ch3: Char = '狐';
```

Comment:
Can this be read-back correctly by the parser? This symbol is not encoded as a Unicode code-point.

Reply:
Fixed